packages/asc-ui/src/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,152 @@
+import Heading from '../Heading'
+import Paragraph from '../Paragraph'
+
+const CardContentStyled = styled(CardContent)`
+  padding-top: 8px;
+  padding-bottom: 8px;
+`
+
+storiesOf('Composed/Card', module)
+  .addDecorator(storyFn => (
+    <div style={{ padding: '40px 10px', background: 'rgb(248,248,248,0.8)' }}>
+      {storyFn()}
+    </div>
+  ))
+  .add('default', () => (
+    <Card maxWidth={250}>
+      <CardContent>
+        <Heading as="h6">This is a card</Heading>
+        <Paragraph>Lorem ipsum dolor sit amet</Paragraph>
+      </CardContent>
+    </Card>
+  ))
+  .add('default with action', () => (
+    <Card maxWidth={250}>
+      <CardContent>
+        <Heading as="h6">This is a card</Heading>
+        <Paragraph>Lorem ipsum dolor sit amet</Paragraph>
+      </CardContent>
+      <CardActions>
+        <Icon size={15}>
+          <ChevronRight />
+        </Icon>
+      </CardActions>
+    </Card>
+  ))
+  .add('default with action and media item', () => (
+    <Card maxWidth={250}>
+      <CardMedia>
+        <Image
+          src="https://www.amsterdam.nl/publish/pages/900852/220px/38701-180724-mauritskade-brug-nathouden_an-02201460.jpg"
+          alt="stadsloket"
+        />
+      </CardMedia>
+      <CardContent>
+        <Heading as="h6">Title</Heading>
+        <Paragraph>This is a card</Paragraph>
+      </CardContent>
+      <CardActions>
+        <Icon size={15}>
+          <ChevronRight />
+        </Icon>
+      </CardActions>
+    </Card>
+  ))
+  .add('with shadow', () => (
+    <Card maxWidth={250} backgroundColor="level2" shadow>
+      <CardContent>
+        <Heading as="h6">This is a card</Heading>
+        <Paragraph>Lorem ipsum dolor sit amet</Paragraph>
+      </CardContent>
+      <CardActions>
+        <Icon size={15}>
+          <ChevronRight />
+        </Icon>
+      </CardActions>
+    </Card>
+  ))
+  .add('horizontal', () => (
+    <Card maxWidth={450} horizontal>
+      <CardMedia backgroundColor="level2">
+        <Icon size={25}>
+          <Download />
+        </Icon>
+      </CardMedia>
+      <CardContent>
+        <Heading as="h6">This is a card</Heading>
+        <Paragraph>Lorem ipsum dolor sit amet</Paragraph>
+      </CardContent>
+      <CardActions>
+        <Icon size={15}>
+          <ChevronRight />
+        </Icon>
+      </CardActions>
+    </Card>
+  ))
+  .add('horizontal - implementation on Dataportaal', () => (
+    <Card maxWidth={450} horizontal>
+      <CardMedia backgroundColor="level2">
+        <Icon size={25}>
+          <Download />
+        </Icon>
+      </CardMedia>
+      <CardContentStyled>
+        <Heading as="h6">This is a card</Heading>
+        <Paragraph>Lorem ipsum dolor sit amet</Paragraph>
+      </CardContentStyled>
+      <CardActions>
+        <Icon size={15}>
+          <ChevronRight />
+        </Icon>
+      </CardActions>
+    </Card>
+  ))
+  .add('loading state - default', () => (
+    <Card maxWidth={250} loading>
```

Comment:
the height of this one is a bit small? Perhaps give a min-height to the Card component? Otherwise the page always jump / change height after loading the content

Reply:
True. I've set a min-height of `90px` to make the `default loading Card` equal to the size of the `rendered default Card`